pupil-spa/src/app/login-success/login-success.component.html

```diff
@@ -13,7 +13,7 @@ <h1 class="heading-xlarge">Welcome, {{pupil.firstName}}</h1>
     <p class="lede">
       <a routerLink="/sign-out" >This is not me, I will enter my details again</a>
     </p>
-    <form action="/check/start" method="GET">
+    <form (ngSubmit)="onSubmit()">
```

Comment:
Now that this is an angular component we probably don't need a `form` here - could just be a button.